output-failures-and-bottle-result/action.yml

```
@@ -0,0 +1,17 @@
+name: 'Output failures summary and brew bottle result'
+description: 'Output the logs of failures summary and brew bottle result'
```

Since these 2 are practically the same, why not drop the description or make it like
```suggestion
name: 'failures summary and bottle result'
description: 'Output the logs of failures summary and brew bottle result'
```

---

I think these two fields are required. I might change it to your suggestion. Might as well to change the directory name.